### PR TITLE
add dns support

### PR DESCRIPTION
**Pull request by markry-kainos**
We want to include a frontend deployment here: https://github.com/hmcts/cmc-claim-store/tree/master/charts/cmc-claim-store

Motivation is so we can test any claim-store changes in isolation, prior to merge.

Primary, pipeline DNS is automatically setup for the claim-store chart. This will add DNS for the frontend too.

### CODE REVIEW

**Comment by markry-kainos** [comment]
tested : namespace = `money-claims`, helm deployment: `cmc-claim-store-pr-681`

example url : https://www-cmc-claim-store-pr-681.service.core-compute-preview.internal/health

File: nodejs/templates/dns.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      containers:
+        - name: {{ template "hmcts.releaseName" . }}-dns
+          image: hmcts/curl
+          volumeMounts:
+            - name: container-init
+              mountPath: /container.init
+          command: [ "/bin/sh" ]
+          args:
+            - /container.init/nodejs-dns
+            - {{ .Values.registerAdditionalDns.prefix }}-{{ .Release.Name }}
```
Comment (luigibk) [inline nodejs/templates/dns.yaml]: Shouldn't there be a check for this to be less than 63 characters (DNS spec)? 
Something like (I think): {{- printf "%s-%s" .Values.registerAdditionalDns.prefix  .Release.Name  | trunc 63 | trimSuffix "-" -}}

Comment (markry-kainos) [reply nodejs/templates/dns.yaml]: this is just an arg though - doesn't it just create an entry with consul (not a k8s resource - where the trunc would be needed)?

Comment (luigibk) [reply nodejs/templates/dns.yaml]: It's a DNS limit, which k8s uses for its resources as well. Probably not an issue in our case. Something to keep in mind though

**Comment by markry-kainos** [comment]
@timja @luigibk pls merge :-)

**Pull request merged**